### ISSUE TITLE
Training scripts

### ISSUE THREAD

**Issue by VGrondin** [facebookresearch/hiera#28]
Hello great paper,

Any idea on when the training files will be available?

**Comment by dbolya** [comment]
Training code kinda fell by the wayside (we have to rewrite it from scratch because the original project used internal training code). I do however plan to get it out by the end of the month alongside the model / code release for the `Window Attention is Bugged` paper.

**Comment by VGrondin** [comment]
Thanks! Looking forward to it :)

**Comment by tkhe** [comment]
hi, is there any new developments？

**Comment by dbolya** [comment]
> hi, is there any new developments？

The in progress training code is available here:
https://github.com/dbolya/hiera/tree/v0.2.0

See `examples/train.py` to see how to use it. However, I'm still waiting on some full training runs to fully test it.

**Comment by tapohongchen** [comment]
Hello, thanks for your work. 
I saw the train in the picture. Any idea on When the train in the video will be available, Especially in the ava.

**Comment by qianyizhang** [comment]
> Training code kinda fell by the wayside (we have to rewrite it from scratch because the original project used internal training code). I do however plan to get it out by the end of this month alongside the model / code release for the `Window Attention is Bugged` paper.

nice work, i would also like too read about HieraDet release :P

**Comment by guillembraso** [comment]
Hi! Is there any update on the release of the training code? Or could someone please share some script with the preliminary release? Thanks in advance! 🙏

**Comment by andyqmongo** [comment]
> > hi, is there any new developments？
> 
> The in progress training code is available here: https://github.com/dbolya/hiera/tree/v0.2.0
> 
> See `examples/train.py` to see how to use it. However, I'm still waiting on some full training runs to fully test it.

Thanks for sharing the training script! However, I wonder why increasing masking ratio doesn't reduce training time. I thought that the work introduced a way to speed up MAE pretrained without compromising quality. Or did I miss something?
Thanks! 🙏

**Comment by guokeqianhg** [comment]
> Training code kinda fell by the wayside (we have to rewrite it from scratch because the original project used internal training code). I do however plan to get it out by the end of this month alongside the model / code release for the `Window Attention is Bugged` paper.

Hi, Thanks for your works. Is the code for  `Window Attention is Bugged` paper released now?
